lang/en.yml

```diff
@@ -8,7 +8,7 @@ en:
     PLURALNAME: 'Broken External Page Tracks'
     SINGULARNAME: 'Broken External Page Track'
   BrokenExternalPageTrackStatus:
-    PLURALNAME: 'Broken External Page Track Statuss'
+    PLURALNAME: 'Broken External Page Track Statuses'
```

Comment:
This is an automatically generated string that gets set when you run the i18nTextCollectorTask. I think you'll need to update the `BrokenExternalPageTrackStatus` model somewhere in order to configure how that pluralises itself - see https://docs.silverstripe.org/en/4/developer_guides/i18n/#pluralisation